tasks/4.js

```diff
@@ -0,0 +1,25 @@
+var arr = [1, 2, 1, 3, 4];
```

Comment:
This does not seem to belong here. 🤔 
Also, consider looking into `Set`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set